src/utils/validatePassport.ts

```diff
@@ -0,0 +1,15 @@
+import { ERROR_MESSAGE } from "../context/alert";
+
+const validate = (inputPassport: string): boolean => {
+  const passportRegex = "^[A-Za-z]{1,3}-[a-zA-Z0-9]{5,9}$";
```

Comment:
erm.. its because we treat this identifier like `NRIC`. Its a known identifier type unlike `STRING` where we provide the `validationRegex`. Just wonder if its weird if we have a `PASSPORT` identifier with different regex for different campaigns

Reply:
I see I see! Hmm, true, let's leave it then!